scalameta/src/main/scala/scala/meta/Trees.scala

```diff
@@ -71,6 +71,7 @@ package scala.meta {
   }
 
   object Ctor {
+    @branch trait Call extends Term
     @branch trait Ref extends Term.Ref
```

Comment:
So now, since we have `Ctor.Call`, we can do three things:
1) Update the spec for Template to use `..$ctorcalls` instead of `..$exprs`.
2) Update Template.parents to say `Seq[Ctor.Call]`
3) Update Template to require isCtorCall for all Template.parents (oops, this is already done)